src/assert.rs

```diff
@@ -26,6 +29,7 @@ impl default::Default for Assert {
         Assert {
             cmd: vec!["cargo", "run", "--"]
                 .into_iter().map(String::from).collect(),
+            env: Environment::empty(),
```

Comment:
Should the default be
- `empty`
- `inherit` (I presume this is `std::process::Command`'s default and the current behavior)
- `None` (use `std::process::Command`'s default which is the current behavior)